docs/additional_info/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 13.0.0
```

Comment:
why semver major?

Reply:
I'm actually now entirely sure. This is a new feature, not anything breaking, so theoretically we could mark it as 12.x. Still, this is a relatively "big" thing. In any case, this pr is not finalized, I'll need support from someone who could check if any fields are missing from these schemas

Reply:
yeah, I's recomment semver minor for non-breaking incremental additions

Reply:
Okay